src/handlers/cross-account-transfer/handler.ts

```diff
@@ -0,0 +1,105 @@
+import { SQSClient, SendMessageBatchCommand } from '@aws-sdk/client-sqs';
+import { S3Client, ListObjectsV2Command, GetObjectCommand } from '@aws-sdk/client-s3';
+import { createGunzip } from 'zlib';
+import { v4 as uuidv4 } from 'uuid';
```

Comment:
do `zlib` and `uuid` need to be installed as a dev dependency?

Reply:
`uuid` is present as a dev dependency. just added the library `zlib` in there. thanks!